test/es6/dynamic-module-import-syntax.js

```diff
@@ -0,0 +1,83 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// ES6 Generators Syntax tests -- verifies function* and yield syntax spec conformance
```

Comment:
This file was redundant and was removed.

---
In reply to: [115339634](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r115339634) [](ancestors = 115339634)